scripts/sql/dbsetup.sql

```diff
@@ -139,6 +140,7 @@ CREATE TABLE fhir_endpoints_info_history (
     validation              JSONB,
     included_fields         JSONB,
     supported_resources     VARCHAR(500)[],
+    response_time_seconds   DECIMAL(64,4),
```

Comment:
why was decimal chosen instead of double precision, out of curiosity?

Reply:
The decimal type has a storage size equal to it's precision, and so with a precision of 64 bits, the storage size for the decimal type in this case is the same as double precision. Therefore, decimal and double precision are essentially equivalent in this case and really either could be used. However, decimal allows for a simple way to scale the data so it only shows a specific amount of significant digits, in this case 4, which is nice for making the response times easier to read in the database, so that was the advantage of decimal that caused me to chose a decimal type over double precision

Reply:
ah okay, that makes sense, thanks!